app/models/post.rb

```diff
@@ -230,10 +230,6 @@ def deleted?
     deletion_logs.where(is_deleted: true).any?
   end
 
-  def conflicted?
-    is_tp && (is_fp || is_naa)
-  end
-
```

Comment:
This method is never used.

Reply:
Worth keeping around, though - potentially useful and not doing any harm.